### ISSUE TITLE
Problem with `bash ls -i` ( -l option is OK, -i option fails )

### ISSUE THREAD

**Issue by oOosys** [geophile/marcel#23]
I am currently working towards a concept of user interaction with the system ( https://github.com/oOosys/oOo ) which seems to share some of main thoughts with the marcel approach to shell scripting. 

Reading about marcel shell I had thought I have understood the approach the right way, but it seems I haven't:

``` bash
~ $ echo $0
bash
~ $ ls -i *.txt
9311566 tmp.txt
```

```marcel
/home/o $ bash ls -l *.txt
-rw-rw-r-- 2 o o 101 2023-09-16 18:49 tmp.txt
/home/o $ bash ls -i *.txt
Operator bash: Flags must all appear before the first anonymous arg
/home/o $ version
0.18.4
/home/o $ ls *.txt
-rw-rw-r--   o      o           101   2023 Sep 16 18:49:37 tmp.txt
/home/o $ ls -i *.txt
Operator ls: Unknown flag -i
/home/o $
```

I have thought that the `bash` operator will take the shell command as it is and pass it to the bash shell ...

Any hints towards better understanding why I have failed to obtain the inode value for the file? Or have I run into a bug?

Claudio ( https://github.com/oOosys/oOo )

**Comment by geophile** [comment]
The problem is that the bash operator has a -i flag (run "help bash" to find out more about it). Argument parsing for bash is different from that of operators, and I got it wrong. Working on a fix.

**Comment by geophile** [comment]
Fixed in 0.18.5.
